crates/bevy_ui/src/lib.rs

```diff
@@ -166,6 +166,7 @@ impl Plugin for UiPlugin {
             (
                 check_visibility::<WithNode>.in_set(VisibilitySystems::CheckVisibility),
                 update_target_camera_system.in_set(UiSystem::Prepare),
+                #[cfg(feature = "bevy_text")]
```

Comment:
```rust
                #[cfg(feature = "bevy_text")]
                ui_layout_system
                    .in_set(UiSystem::Layout)
                    .before(TransformSystem::TransformPropagate)
                    // Text and Text2D operate on disjoint sets of entities
                    .ambiguous_with(bevy_text::update_text2d_layout),
```

I might be missing something, but after rebasing, it seemed like bevy_text usage was not conditionally set here like it was everywhere else.

Reply:
This feature flag needs to be moved to the `.ambiguous_with(bevy_text::update_text2d_layout)` line.

Reply:
Ah shoot, I totally glossed over that, thinking this was just something specific to `bevy_text`

I think covers it since you cant conditionally chain a method afaik https://github.com/bevyengine/bevy/pull/12803/commits/06dd73c4c89b703b703a451c88d59250b8c255c5